tauri-app/src/routes/orders/[network]-[orderHash]/+page.svelte

```
@@ -4,17 +4,80 @@
+    handleDebugTradeModal,
+    handleQuoteDebugModal,
+    handleDepositModal,
+    handleWithdrawModal,
+  } from '$lib/services/modal';
   import type { Hex } from 'viem';
+  import type { SgVault } from '@rainlanguage/orderbook/js_api';
+  import { Toast } from 'flowbite-svelte';
+  import { CheckCircleSolid } from 'flowbite-svelte-icons';
+  import { fade } from 'svelte/transition';
+  import { useQueryClient } from '@tanstack/svelte-query';
+  import { walletconnectAccount } from '$lib/stores/walletconnect';
```

ESLint

[error] 'walletconnectAccount' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)